hknweb/events/views.py

```diff
@@ -47,9 +48,7 @@ def unrsvp(request, id):
         raise Http404()
 
     event = Event.objects.get(pk=id)
-    if request.user.is_authenticated and event.rsvps < event.rsvp_limit:
-        #check if rsvp for this event and this user already exists; if false, then set true
-        event.rsvps = F("rsvps") - 1
+    if request.user.is_authenticated:
         Rsvp.objects.get(user=request.user, event=event).delete()
```

Comment:
This will throw a [`Model.DoesNotExist` error](https://docs.djangoproject.com/en/2.2/ref/models/instances/#django.db.models.Model.DoesNotExist) if the Rsvp object does not exist; perhaps handle this exception?